docs/_sass/custom/custom.scss

```diff
@@ -0,0 +1,8 @@
+div.site-header {
+  max-height: 15em;
+  height: 15em;
```

Comment:
As `max-height` attribute is set in theme styling found I needed to override both `max-height` and `height` for this to work but someone with more CSS knowledge may know a better approach.